bin/mutePublicSquadNotifications.ts

```diff
@@ -0,0 +1,68 @@
+import fastq from 'fastq';
+import '../src/config';
+import createOrGetConnection from '../src/db';
+import { Source, SourceMember } from '../src/entity';
+import {
+  NotificationPreferenceStatus,
+  NotificationType,
+  saveNotificationPreference,
+} from '../src/notifications/common';
+
+interface SourceMemberQueue {
+  userId: string;
+  sourceId: string;
+}
+
+const QUEUE_CONCURRENCY = 1;
+
+(async () => {
```

Comment:
I think we should not apply it retroactively as some users are already used to getting this notifications. Wdyt @sshanzel @rebelchris ?

Reply:
Hmm that doesn't solve all existing users getting annoyed by it? (including me and you 😂)
Why would we not do it retro?

Reply:
Do it retroactively, but mention it in a changelog or something, so people know _why_ they might not get the notifications anymore